### DISCUSSION TITLE
I am start destack in my next js i found new err

### DISCUSSION THREAD

**Discussion by chandranfullstack** [LiveDuo/destack#88] [Q&A]
Unhandled Runtime Error
TypeError: Path must be a string. Received undefined

Call Stack
assertPath
node_modules\next\dist\compiled\path-browserify\index.js (1:95)
Object.parse
node_modules\next\dist\compiled\path-browserify\index.js (1:4265)
eval
node_modules\destack\build\server\index.js (1886:0)
(app-client)/./node_modules/destack/build/server/index.js
file:///C:/Users/ravic/Desktop/Flugid/nextbuilder/.next/static/chunks/app/page.js (105:1)
options.factory
/_next/static/chunks/webpack.js (704:31)
__webpack_require__
file:///C:/Users/ravic/Desktop/Flugid/nextbuilder/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (359:21)
eval
webpack-internal:///(app-client)/./src/app/page.js (7:78)
(app-client)/./src/app/page.js
file:///C:/Users/ravic/Desktop/Flugid/nextbuilder/.next/static/chunks/app/page.js (159:1)
options.factory
/_next/static/chunks/webpack.js (704:31)
__webpack_require__
file:///C:/Users/ravic/Desktop/Flugid/nextbuilder/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (359:21)
Next.js
requireModule
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (144:0)
initializeModuleChunk
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1205:0)
resolveModuleChunk
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1163:0)
eval
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1590:0)

**Comment by LiveDuo** [comment]
Closing this as it's now outdated. If you are still facing problems please open an issue instead.
